ConsoleApp/Program.cs

```diff
@@ -0,0 +1,95 @@
+﻿static bool StringCompare(string str1, string str2) => str1 == str2;
+static (int,int,int)StringAnalyze(string str)
+{
+    int digits = 0, alphabetic = 0, other = 0;
+    foreach (char i in str)
+    {
+        if (char.IsDigit(i))
+        {
+            digits += 1;
+            continue;
+        }
+        if (char.IsLetter(i))
+        {
+            alphabetic += 1;
+            continue;
+        }
+        other++;
```

Comment:
if char is not letter or digit - it does not mean it's special character

Reply:
Done!